src/mmw/apps/geoprocessing_api/calcs.py

```diff
@@ -226,8 +231,10 @@ def point_source_pollution(geojson):
             point_source_results = []
 
     return {
-        'displayName': 'Point Source',
-        'name': 'pointsource',
+        'displayName': 'Point Source ({})'.format(datasource.upper())
+        if datasource else 'Point Source',
+        'name': 'pointsource ({})'.format(datasource.upper())
```

Comment:
No spaces allowed here.

```suggestion
        'name': f'pointsource_{datasource}'
```